### ISSUE TITLE
IsAuthorOfPublication presents two relationships (person & orgUnit)

### ISSUE THREAD

**Issue by YPaulsen-TLC** [DSpace/DSpace#9313]
**Describe the bug**
When I set up a current standard installation of DSpace and activate the entities, I have a bug with the relationship isAuthorOfPublication/isPublicationOfAuthor. I link an item with the entity type Publication with an item of the entity type Person. If I now edit the publication and go to the "Relationships" tab, I see the person listed twice. Once under "Authors (persons)" and once under "Authors (organizational units)". Both entities (persons and orgUnits) are using the identical relationship in the relationship-types.xml. 
I have the same problem at https://demo.dspace.org/home.

<img width="1145" alt="Bildschirmfoto 2024-02-08 um 15 44 51" src="https://github.com/DSpace/DSpace/assets/83951759/6abada70-8681-44e9-ad71-725f2daf8e28">

**Comment by uofmsean** [comment]
This looks like a duplicate of #9115

**Comment by tdonohue** [comment]
Thanks for pointing that out, @uofmsean !   Closing this as it's definitely a duplicate of #9115.   #9115 is waiting on a volunteer to help resolve the bug.
